docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_managed_identity/linux/_index.md

```diff
@@ -0,0 +1,405 @@
+    ```shell
+    challengeTokenPath=$(curl -s -D - -H Metadata:true "http://127.0.0.1:40342/metadata/identity/oauth2/token?api-version=2019-11-01&resource=https%3A%2F%2Fmanagement.azure.com" | grep Www-Authenticate | cut -d "=" -f 2 | tr -d "[:cntrl:]")
+    ```
+
+
+    The filepath is included in the headers returned by the REST API call.
+
+    ![Screenshot showing the command output](./12.png)
+
+    The file is created in a path that only the root and himds IDs can access.
+
+
+1. Read the content of the file into a variable
+
+    ```shell
+    challengeToken=$(sudo cat $challengeTokenPath)
+    ```
+
+    The challenge token files may only be used once and are automatically removed.
+
+    ![Screenshot showing the command output](./13.png)
+
+1. Access the identity endpoint to acquire the resource token
+
+    ```shell
+    token=$(curl -s -H Metadata:true -H "Authorization: Basic $challengeToken" "http://127.0.0.1:40342/metadata/identity/oauth2/token?api-version=2019-11-01&resource=https%3A%2F%2Fmanagement.azure.com" | jq -r .access_token)
+    ```
+
+    Note the resource value in the query and the challenge token in the header.
+
+    ![Screenshot showing the command output](./14.png)
+
+    The resulting token is valid for 24 hours.
+
+## JWT Token
+
+* The resulting token may be pasted into <https://jwt.ms> to decode it and view the claims.
+
+  ![https://jwt.ms](./15.png#bordered)
+
+## Accessing the ARM REST API
+
+* List out the tags used in the subscription
+
+    This command is purely to demonstrate that the Azure Resource Manager API can be accessed by the trusted compute.
+
+    ```shell
+    curl -sSL -X GET -H "Authorization: Bearer $token" -H "Content-Type: application/json" https://management.azure.com/subscriptions/$subscriptionId/tagNames?api-version=2021-04-01 | jq .value[].tagName
+    ```
+
+    ![Screenshot showing the command output](./16.png)
+
+
+    This REST API call will fail if you have not assigned the Reader role to the subscription scope for the managed identity.
+
+## Accessing Key Vault secrets
+
+You will access other endpoints when using PaaS services such as Azure Key Vault, Azure Storage Account, Azure PaaS databases etc.
+
+The token acquisition process is the same except you have to change the resource (or audience) in the query of the call.
+
+* Acquire a token for the Key Vault service
+
+    ```shell
+    challengeTokenPath=$(curl -s -D - -H Metadata:true "http://127.0.0.1:40342/metadata/identity/oauth2/token?api-version=2019-11-01&resource=https%3A%2F%2Fvault.azure.net" | grep Www-Authenticate | cut -d "=" -f 2 | tr -d "[:cntrl:]")
+    challengeToken=$(sudo cat $challengeTokenPath)
+    vaultToken=$(curl -s -H Metadata:true -H "Authorization: Basic $challengeToken" "http://127.0.0.1:40342/metadata/identity/oauth2/token?api-version=2019-11-01&resource=https%3A%2F%2Fvault.azure.net" | jq -r .access_token)
+    ```
+
+    > Note that the resource is set to `https://vault.azure.net` in the query.
```

Comment:
This note should be positioned before the code block. Below the code block, there should be a screenshot highlighting this.